docs/content/preview/migrate/yb-voyager/perform-migration.md

```diff
@@ -0,0 +1,148 @@
+
+{{< note title="Note" >}}
+
+The `yb-voyager import schema` command does not import indexes yet. This is done to speed up the data import phase. The indexes will be created by `yb-voyager import data` command after importing the data in the next step.
+
+{{< /note >}}
+
+## Import data
+
+After you have successfully exported the source data and imported the schema in the target database, you can now import the data using the `yb-voyager import data` command:
+
+```sh
+yb-voyager import data --export-dir ${EXPORT_DIR} \
+        --target-db-host ${TARGET_DB_HOST} \
+        --target-db-port ${TARGET_DB_PORT} \
+        --target-db-user ${TARGET_DB_USER} \
+        --target-db-password ${TARGET_DB_PASSWORD:-''} \
+        --target-db-name ${TARGET_DB_NAME}
+```
+
+In the import data phase, yb-voyager splits the data dump files (from the `$EXPORT_DIR/data` directory) into smaller _batches_ ,each of which contains at most `--batch-size` number of records. yb-voyager concurrently ingests the batches such that all nodes of the target YugabyteDB cluster are utilized. This phase is designed to be _restartable_ if yb-voyager terminates when the data import is in progress. Upon a restart, the data import resumes from its current state.
+
+The `yb-voyager import data` command reads data files located in the `EXPORT_DIR/data`. The command, by default, creates one database connection to each of the nodes of the target YugabyteDB cluster. You can increase the number of connections by specifying the total connection count in the `--parallel-jobs` argument. The command will equally distribute the connections among all the nodes of the cluster. It splits the larger tables into smaller chunks, each containing at most `--batch-size` number of records. By default, the `--batch-size` is 100,000 records.
+
+Run the `yb-voyager import data status --export-dir ${EXPORT_DIR}` command to get an overall progress of the data import operation. While importing a very large database, you should run the import data command in a `screen` session, so that the import is not terminated when the terminal session stops. If the `yb-voyager import data` command terminates before it could complete the data ingestion, you can re-run it with the same arguments and the command will resume the data import operation.
+
+### Import data file
+
+The `yb-voyager import data file` command is an alternative to the `yb-voyager import data` command, using which you can load the data into the target table directly from the file(s). This command doesn’t require performing other phases of migration :[export and analyze schema](#export-and-analyze-schema), [export data](#export-data), or [import schema](#import-schema), prior to the import data phase. It only requires a table present in the target database to perform the import.
```

Comment:
`import data file` is not an "alternative" to `import data`. They both are useful in different cases.

Reply:
Could you please specify the use cases @amit-yb . I am re-ordering the import data phase. Please take a look.

Reply:
Thank you.